Libraries/Alert/Alert.js

```diff
@@ -24,6 +24,23 @@ export type Buttons = Array<{
   style?: AlertButtonStyle,
   ...
 }>;
+export type AlertKeyboardType =
+  // Cross Platform
+  | 'default'
+  | 'email-address'
+  | 'numeric'
+  | 'phone-pad'
+  | 'number-pad'
+  | 'decimal-pad'
+  | 'url'
+  // iOS-only
+  | 'ascii-capable'
+  | 'numbers-and-punctuation'
+  | 'name-phone-pad'
+  | 'twitter'
+  | 'web-search'
+  // Android-only
+  | 'visible-password';
```

Comment:
For these platform specific types, can you add them in platform specifc type files so that it's a bit safer to keep track of them? e.g.

Put types in mytypes.android.js, mytypes.ios.js and import {type MyType} from 'mytypes';

Reply:
Makes sense. Will add that in